### PR TITLE
Create/Crypto/Route

### PR DESCRIPTION
**Pull request by siasktv**
Create/Crypto Route

### CODE REVIEW

File: server/controllers/cryptoController.js
Comment (amberhunt955) [inline server/controllers/cryptoController.js:1]: So I think technically the comment on line 3 is incorrect - you ARE fetching the data, but that data is then being populated to the preexisting database. The database is not being created here. But your code is good!

File: server/package.json
Comment (amberhunt955) [inline server/package.json:1]: For some reason axios is missing from the dependencies - so I went ahead and added it.

File: server/routes/cryptoRoutes.js
Comment (amberhunt955) [inline server/routes/cryptoRoutes.js:1]: So this should actually be a get route instead of a post route - I couldn't get it to populate the DB otherwise.

**Review by amberhunt955 — APPROVED** [review]
Just a few slight changes made, otherwise great work on this!

File: server/controllers/cryptoController.js
Comment (siasktv) [reply server/controllers/cryptoController.js:1]: Hi Amber! thank you so much for reviewing the code! The create/crypto route is actually to create documents on MongoDBwith the crypto API information provided by the url. I tried the route and it populates all the information about cryptos on the MongoDB it creates and populates.  You can try the route on postman with a POST create/crypto to create the crypto info on MongoDB :) hope it helps!

File: server/routes/cryptoRoutes.js
Comment (siasktv) [reply server/routes/cryptoRoutes.js:1]: Hi Amber! I still need to work on the get route, but this reoute creates the cryptos info on MongoDB. You can try the route using POST on postman at create/crypto

Comment (amberhunt955) [reply server/routes/cryptoRoutes.js:1]: Ah yes, sorry, you are right about this! My bad, maybe this is another reason why I shouldn't look at code so late at night 😂

**Pull request merged**